server/models/trip.js

```diff
@@ -3,33 +3,34 @@ const mongoose = require('mongoose');
 const tripSchema = mongoose.Schema({
   title: {
     type: String,
-    required: true,
+    required: [true, 'Title is required'],
```

Comment:
The next step is to implement the same feedback as it is already implemented in the Login or SignUp page